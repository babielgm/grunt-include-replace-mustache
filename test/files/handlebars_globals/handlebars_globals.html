@@include('handlebars.html',{
"substantive": "sloth",
"verb": "hanging",
"object": "from the tree"
})