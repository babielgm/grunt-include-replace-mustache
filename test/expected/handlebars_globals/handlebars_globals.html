The sloth is hanging from the tree and has curly hair.